File: 0004-median-of-two-sorted-arrays/0004-median-of-two-sorted-arrays.cpp
class Solution {
public:
    double findMedianSortedArrays(vector<int>& nums1, vector<int>& nums2) {
        vector<int> v;
        for(auto it:nums1){
            v.push_back(it);
        }
        for(auto it:nums2){
            v.push_back(it);
        }
        sort(v.begin(),v.end());
        if(v.size()%2==0){
            return (double)(v[v.size()/2]+v[v.size()/2-1])/2;
        }
        return (double)v[v.size()/2];
    }
};

File: 0004-median-of-two-sorted-arrays/0004-median-of-two-sorted-arrays.java
class Solution {
    public double findMedianSortedArrays(int[] nums1, int[] nums2) {
        List<Integer> v = new ArrayList<>();
        for(int i=0;i<nums1.length;i++){
            v.add(nums1[i]);
        }
        for(int i=0;i<nums2.length;i++){
            v.add(nums2[i]);
        }
        Collections.sort(v);
        if (v.size() % 2 == 0) {
            return (double) (v.get(v.size() / 2) + v.get(v.size() / 2 - 1)) / 2;
        }
        return (double) v.get(v.size() / 2);
    }
}